packages/frontend/app/components/reports/subject/new/session-type.gjs

```diff
@@ -31,6 +32,16 @@ export default class ReportsSubjectNewSessionTypeComponent extends Component {
   }
 
   get sortedSessionTypes() {
+    if (!this.args.school) {
+      return sortBy(
+        this.filteredSessionTypes.map((st) => ({
+          id: st.id,
+          title: `${st.school.get('title')}: ${st.title}${!st.active ? ` (${this.intl.t('general.inactive')})` : ''}`,
```

Comment:
`st.school` is a async relationship. i don't see a guarantee that is resolved when accessing the school title here. 
suggesting you wrap this in a async function + TrackedAsyncData in a companion `sortedSessionTypesData` getter and resolve it there.

needs `const school = await st.school;` somewhere in there.